operating/manage-compliance/managing-compliance.adoc

```diff
@@ -0,0 +1,92 @@
+
+* The compliance feature requires the Compliance Operator to be running and does _not_ support Amazon Elastic Kubernetes Service (EKS).
+====
+
+With {product-title-short} you can assess, check and report on the compliance status of your containerized infrastructure.
+
+You can run out-of-the-box compliance scans based on the following industry standards:
+
+* *CIS Benchmarks* (Center for Internet Security) for *Docker* and *Kubernetes*
+* *HIPAA* (Health Insurance Portability and Accountability Act)
+* *NIST Special Publication 800-190* and *800-53* (National Institute of Standards and Technology)
+* *PCI DSS* (Payment Card Industry Data Security Standard)
+* *OpenSCAP* (Open Security Content Automation Protocol): Available in {product-title-short} for {ocp} clusters when the Compliance Operator is installed and configured to deliver results to {product-title-short}
+
+By scanning your environment based on these standards, you can:
+
+* Evaluate your infrastructure for regulatory compliance.
+* Harden your Docker Engine and Kubernetes orchestrator.
+* Understand and manage the overall security posture of your environment
+* Get a detailed overview of the compliance status of clusters, namespaces and nodes.
+
+
+// Viewing the compliance standards across your environment
+include::modules/viewing-the-compliance-standards-across-your-environment.adoc[leveloffset=+1]
+
+// Checking the compliance status of your infrastructure
+include::modules/checking-the-compliance-status-of-your-infrastructure.adoc[leveloffset=+1]
+
+// Compliance dashboard overview
+include::modules/compliance-dashboard-overview.adoc[leveloffset=+1]
+
+// Viewing the compliance status for clusters
+include::modules/viewing-the-compliance-status-for-clusters.adoc[leveloffset=+2]
+
+// Viewing the compliance status for namespaces
+include::modules/viewing-the-compliance-status-for-namespaces.adoc[leveloffset=+2]
+
+// Viewing the compliance status for a specific standard
+include::modules/viewing-the-compliance-status-for-a-specific-standard.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../operating/manage-security-policies.adoc#manage-security-policies[Managing security policies]
+
+// Viewing the compliance status for a specific control
+include::modules/viewing-the-compliance-status-for-a-specific-control.adoc[leveloffset=+2]
+
+// Limiting the amount of data visible in the compliance dashboard
+include::modules/limiting-the-amount-of-data-visible-in-the-compliance-dashboard.adoc[leveloffset=+1]
+
+// Tracking the compliance status of your environment
+include::modules/tracking-the-compliance-status-of-your-environment.adoc[leveloffset=+1]
+
+// Evidence reports
+include::modules/evidence-reports.adoc[leveloffset=+2]
+
+// Supported benchmark versions
+include::modules/supported-benchmark-versions.adoc[leveloffset=+1]
+
+//Assessing the profile compliance across clusters
```

Comment:
Everything below this is in tech preview.  Everything above this line is simply relocated V1 stuff.  The whole section is listed as tech preview which is not quite accurate.
![image](https://github.com/openshift/openshift-docs/assets/99685630/6b16544a-4aad-4778-9a2e-32d4951880f8)